### ISSUE TITLE
Version 2.0 fails on archive

### ISSUE THREAD

**Issue by jeremiahthomasharris** [SurveyMonkey/surveymonkey-ios-sdk#43]
I receive this error when archiving the build:

```
ld: bitcode bundle could not be generated because '/Users/xxx/xxx/xxx/xxx/Pods/surveymonkey-ios-sdk/SurveyMonkeyiOSSDK.framework/SurveyMonkeyiOSSDK(SMNetworkUtil.o)' was built without full bitcode. All object files and libraries for bitcode must be generated from Xcode Archive or Install build file '/Users/xxx/xxx/xxx/xxx/Pods/surveymonkey-ios-sdk/SurveyMonkeyiOSSDK.framework/SurveyMonkeyiOSSDK' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by benleiken** [comment]
@jeremiahthomasharris sincere apologies for the delay -- I just released 2.0.1 -- let me know if that fixes your issue.

**Comment by jeremiahthomasharris** [comment]
Its resolved and building. Thanks!
